spec/spec_helper_acceptance.rb

```diff
@@ -41,6 +50,21 @@
   end
 end
 
+unless ENV['RS_PROVISION'] == 'no' or ENV['BEAKER_provision'] == 'no'
+
+  install_pe if ENV['BEAKER_TESTMODE'] == 'agent'
+  install_puppet_agent_on default if ENV['BEAKER_TESTMODE'] == 'apply'
```

Comment:
Writing it like this, the test suite will not pick up the CI configuration we have in place. I guess this is what we were talking about the requirement to integrate testmode_switcher and the puppet_install_helper?